test/test_transforms.py

```diff
@@ -205,7 +206,10 @@ def test_mfcc(self):
 
         self.assertTrue(torch_mfcc_norm_none.allclose(norm_check))
 
-    @unittest.skipIf(not IMPORT_LIBROSA or not IMPORT_SCIPY, 'Librosa and scipy are not available')
+    @unittest.skipIf(
```

Comment:
for future refrence: You can use "unittest.skip" as well and don't need a global